src/hooks/persisted/useNovel/useNovelData.ts

```
@@ -0,0 +1,223 @@
+
+    if (novel && page) {
+      let newChapters: ChapterInfo[] = [];
+
+      const config = [novel.id, settingsSort, settingsFilter, page] as const;
+
+      let chapterCount = await getChapterCount(novel.id, page);
+      if (chapterCount) {
+        try {
+          newChapters = (await getPageChaptersBatched(...config)) || [];
+        } catch (error) {
+          console.error('Error fetching chapters:', error);
```

ESLint

[error] Unexpected console statement.

(no-console)